<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proxy</class>
 <widget class="QWidget" name="Proxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>574</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>756</width>
    <height>574</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>756</width>
    <height>574</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proxy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="StackedWidget">
     <widget class="QWidget" name="Page1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Proxy</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Auto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="autoCheckBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="hideCheckBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWidget" name="autoWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>AutoURL</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="urlLineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="minimumSize">
            <size>
             <width>560</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>560</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Manual</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="manualCheckBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QWidget" name="manualWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>560</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>560</width>
             <height>200</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Http Proxy</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Https Proxy</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Ftp Proxy</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Socks Proxy</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="hostVLayout">
                <item>
                 <widget class="QLineEdit" name="httpLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="httpsLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ftpLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="socksLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="portVlLayout">
                <item>
                 <widget class="QLineEdit" name="httpportLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="httpsportLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ftpportLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="socksportLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Cetification</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>List of ignored hosts. more than one entry, please separate with english semicolon(;)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="ignorehostTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>560</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
